build.gradle.kts

```diff
@@ -246,31 +247,6 @@ spotless {
     }
 }
 
-tasks.register("aggregateJavadocs", Javadoc::class.java) {
-    setDestinationDir(project.layout.buildDirectory.file("docs/javadoc").get().asFile)
```

Comment:
This would previously put all the javadocs from all subprojects in the root which meant a lot of the files were overwritten like `index.html`.